google-cloud-pubsub/docs/toc.json

```diff
@@ -52,12 +52,12 @@
           "patterns": ["\/pubsub\/v1"],
           "nav": [
             {
-              "title": "PublisherApi",
-              "type": "google/cloud/pubsub/v1/publisherapi"
+              "title": "PublisherClient",
+              "type": "google/cloud/pubsub/v1/publisherclient"
             },
             {
-              "title": "SubscriberApi",
-              "type": "google/cloud/pubsub/v1/subscriberapi"
+              "title": "SubscriberClient",
+              "type": "google/cloud/pubsub/v1/subscriberclient"
             },
```

Comment:
Can you copy these changes to [google-cloud/docs/toc.json](https://github.com/swcloud/google-cloud-ruby/blob/0fa6e244734788c76a966ccbf3bf2000c3022ec7/google-cloud/docs/toc.json#L338-L345) as well please?